.npmignore

```diff
@@ -0,0 +1,91 @@
+# JS
+node_modules
+yarn.lock
+
+# Project files
+CONTRIBUTORS.md
+CONTRIBUTING.md
+CODE_OF_CONDUCT.md
+
+# Config files
+.babelrc
+babel.config.js
+.editorconfig
+.eslintrc
+.flowconfig
+.watchmanconfig
+jsconfig.json
+.npmrc
+.gitattributes
+.circleci
+*.coverage.json
+.opensource
+.circleci
+.eslintignore
+codecov.yml
+
+# Scripts
+scripts/
+
+# Example
+img/
+example/
+app.json
+
+# Android
+android/*/build/
+android/gradlew
+android/build
+android/gradlew.bat
+android/gradle/
+android/com_crashlytics_export_strings.xml
+android/local.properties
+android/.gradle/
+android/.signing/
+android/.idea/gradle.xml
+android/.idea/libraries/
+android/.idea/workspace.xml
+android/.idea/tasks.xml
+android/.idea/.name
+android/.idea/compiler.xml
+android/.idea/copyright/profiles_settings.xml
+android/.idea/encodings.xml
+android/.idea/misc.xml
+android/.idea/modules.xml
+android/.idea/scopes/scope_settings.xml
+android/.idea/vcs.xml
```

Comment:
This long list of 10+ specific files can be replaced with a simpler and more inclusive single line

```
android/.idea/
```